src/parsers/native_parser.mly

```diff
@@ -44,7 +44,7 @@
 %token AND LEFTARROW RIGHTARROW AC AT AXIOM CASESPLIT REWRITING
 %token BAR HAT
 %token BOOL COLON COMMA PV DISTINCT DOT SHARP ELSE OF EOF EQUAL
-%token EXISTS FALSE VOID FORALL FUNC GE GOAL GT CHECK CUT
+%token EXISTS FALSE VOID FORALL FUNC GE CHECK_VALID CHECK_SAT GT CHECK CUT
```

Comment:
Iirc, we have some kind of doc for the native ae language somewhere (in a sphinx doc I think ?), and so, this change to the language should also change what little doc we have for it, ^^